scapy/asn1/asn1.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, key, value, context=None, codec=None):
             codec = {}
         self._codec = codec
 
-    def clone(self):  # /!\ not a real deep copy. self.codec is shared
```

Comment:
Are these changes needed?

Reply:
This PR does not do only ResourceWarning anymore, but all sort of warnings.

`\ ` is detected as an unknown character escape, and thus raises a SyntaxWarning (becomes a syntaxError with the -W option)